src/panels/lovelace/entity-rows/hui-number-entity-row.ts

```diff
@@ -123,7 +123,6 @@ class HuiNumberEntityRow extends LitElement implements LovelaceRow {
   static get styles(): CSSResultGroup {
     return css`
       :host {
-        cursor: pointer;
```

Comment:
This seems wrong as it is giving a pointer to the entire row, but only certain elements within the row were actually clickable (info div and icon). So some places had a pointer but were not clickable. generic entity row already handles the cursor for the clickable elements, so don't think this is needed.